src/pages/Swap/index.tsx

```diff
@@ -475,271 +539,245 @@ export default function Swap({ className }: { className?: string }) {
   const showDetailsDropdown = Boolean(
     !showWrap && userHasSpecifiedInputOutput && (trade || routeIsLoading || routeIsSyncing)
   )
+  const disableTokenInputs = chainId !== connectedChainId
```

Comment:
when would this happen on the swap page? isn't this only relevant on token details?

Reply:
that's correct. i can move it out to the callsite